crates/nu-command/src/experimental/scope_hack.rs

```diff
@@ -0,0 +1,71 @@
+use nu_protocol::ast::Call;
+use nu_protocol::engine::{Command, EngineState, Stack};
+use nu_protocol::{
+    Category, Example, IntoPipelineData, PipelineData, ShellError, Signature, StateWorkingSet, SyntaxShape, Type,
+    Value,
+};
+
+// command to manipulate scope ($nu.scope) for a particular function
+// because custom commands don't populate signature, category and many other scope fields
+// this utility lets you put whatever you want into any field of scope *for any command*.  Use with care.
+
+#[derive(Clone)]
+pub struct ScopeHack;
+
+impl Command for ScopeHack {
+    fn name(&self) -> &str {
+        "scope-hack"
+    }
+
+    fn usage(&self) -> &str {
+        "Manipulate scope for existing defined function."
```

Comment:
```suggestion
        "Manipulate scope for an existing defined function."
```

Reply:
Didn't mean to include that file in this PR at all...